media-libs/ctl/ctl-1.5.2-r1.ebuild

```diff
@@ -7,7 +7,7 @@ inherit cmake-utils
 
 DESCRIPTION="AMPAS' Color Transformation Language"
 HOMEPAGE="https://github.com/ampas/CTL"
-SRC_URI="${HOMEPAGE}/archive/ctl-${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/ampas/CTL/archive/ctl-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
I suppose you could use `${P}.tar.gz` here, and remove the rename-to-self. Wonderful how people use HOMEPAGE but fail to use what really makes sense.